### ISSUE TITLE
multi gpu error

### ISSUE THREAD

**Issue by cswwp** [zhoudaxia233/EfficientUnet-PyTorch#6]
![image](https://user-images.githubusercontent.com/11901683/66640853-4329e700-ec4c-11e9-9fd6-82b843742cdc.png)

It's a strange error when i inference model efficientb5 unet with tensor with shape 4*3*128*800, seems torch.cat([x, blocks.popitem()[1]], dim=1) error,    i print shape  as upper image, but it shows only in some batchs,  i'm sure input tensor shape is all same (batch*3*128*800)  anyone help?

**Comment by zhoudaxia233** [comment]
@NPU-Franklin Franklin created a PR (#11 ) to support multi GPUs. I do not have multi cards therefore I cannot test it. But maybe you can give it a try.
